recipes-wpe/wpebackend-rdk/wpebackend-rdk_0.1.bb

```diff
@@ -7,9 +7,8 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=ab5b52d145a58f5fcc0e2a531e7a2370"
 DEPENDS += "libwpe glib-2.0"
 
 SRC_URI = "git://github.com/WebPlatformForEmbedded/WPEBackend-rdk.git;protocol=http;branch=master \
-           file://0001-include-array-for-std-array.patch \
```

Comment:
Removing this patch as PR https://github.com/WebPlatformForEmbedded/WPEBackend-rdk/pull/31 does that.